docs/abc.rst

```diff
@@ -19,18 +19,18 @@ aiohttp.web is built on top of few concepts: *application*, *router*,
 *router* is a *plugable* part: a library user may build a *router*
 from scratch, all other parts should work with new router seamlessly.
 
-:class:`AbstractRouter` has the only mandatory method:
-:meth:`AbstractRouter.resolve` coroutine. It must return an
-:class:`AbstractMatchInfo` instance.
+:class:`aiohttp.abc.aiohttp.abc.AbstractRouter` has the only mandatory method:
```

Comment:
these lines contain duplicate text

Reply:
:py:class:`aiohttp.abc.aiohttp.abc.AbstractRouter` from https://webknjaz.github.io/intersphinx-untangled/docs.aiohttp.org/

Reply:
Oh, wow. Sounds like there's some bug in how those docs are linked. Okay, leave it be.